test/test_KubeLibrary.py

```diff
@@ -19,6 +19,7 @@ class AttributeDict(object):
     allowing you to recurse down nested dicts (like: AttributeDict.attr.attr)
     """
     def __init__(self, entries):
+        self._root = None
```

Comment:
Why do we need it?

Reply:
We don't need it, but I think is a good practice to keep instance variables in __init__. Even My linter says it's a code smell :)

Boring stackoverflow argumentation: https://stackoverflow.com/questions/38377276/instance-variables-in-methods-outside-the-constructor-python-why-and-how

Reply:
Aaa sorry I was looking at KubeLibrary.py and couldn't fin d it, yeah that's fine.